SEP-0011.md

```diff
@@ -0,0 +1,154 @@
+# SEP-0011 -- Formatting Code with *Black*
```

Comment:
There definitely needs to be an attribution section in this somewhere... :wink:

Reply:
Done